client/js/test/fixtures/sheet-container.html

```diff
@@ -6,9 +6,9 @@
       </tr>
     </thead>
     <tfoot>
-        <tr>
-        </tr>
-      </tfoot>
+      <tr>
```

Comment:
Hi Kay. I am not quite sure I understand what you mean here. I checked the sheet-container.html file and the thead, tfoot, and tbody tags are all below the table tag. Please help me clarify what I should change. Thanks!